package.json

```diff
@@ -192,27 +192,33 @@
           "ui-circulation.settings.edit-circulation-rules"
         ]
       },
+      {
+        "permissionName": "ui-circulation.settings.view-staff-slips",
```

Comment:
Add this new perm name to translations also, plase

Reply:
Thank you, fixed (https://github.com/folio-org/ui-circulation/pull/1127/commits/67792109215c963a8d8bc2b6a7e71ebb612619a3) Could you please rereview?